CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 0.5.3
```

Comment:
No need to change the change log I do that when preparing a release.

Reply:
I've removed my changelog entry.